deb/debian/stackdriver-agent.init

```diff
@@ -30,7 +30,7 @@ DESC="Stackdriver metrics collection agent"
 NAME=stackdriver-agent
 DAEMON=/opt/stackdriver/collectd/sbin/stackdriver-collectd
 
-CONFIG=/opt/stackdriver/collectd/etc/collectd.conf
+CONFIG=/etc/stackdriver/collectd.conf
```

Comment:
Is it worthwhile to consider making this conditional - i.e., that we look in both the new and old locations and use the config file we find in one of those places (with preference for using the new location)?

Reply:
Not supported by collectd (and unnecessary, IMO).

Reply:
I agree with Igor. I'd rather make a hard and fast change and stick to it than support a bunch of possibilities which may confuse users.

Reply:
SGTM.